CMakeLists.txt

```diff
@@ -163,29 +352,17 @@ else()
     )
 endif()
 
-option(BUILD_CXX_SHARED_LIB "Build a Shared Libraries of the CXX interface" OFF)
-mark_as_advanced(BUILD_CXX_SHARED_LIB)
-
 if(
     INTERFACE_BUILD
     OR BUILD_GRIDDYN_SHARED_LIBRARY
     OR BUILD_CXX_SHARED_LIB
-    OR BUILD_SHARED_FMI_LIBRARY
+    OR ENABLE_FMI_EXPORT
 )
     set(BUILD_SHARED_LIBS ON)
-endif()
-cmake_dependent_option(
-    USE_POSITION_INDEPENDENT_CODE
-    "Build the libraries with Position independent code Useful if only building the static library and it will be used later in a shared library"
-    OFF
-    "NOT BUILD_SHARED_LIBS"
-    OFF
-)
-
-if(BUILD_SHARED_LIBS OR USE_POSITION_INDEPENDENT_CODE)
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
we need some way to turn on position independent code if the shared_libraries are not built.  Is this just assuming we use this variable?  if so we need to expose is in some way via mark_as_advanced or some thing of that nature so it shows up as an option.

Reply:
You can do `-DCMAKE_POSITION_INDEPENDENT_CODE=ON` still? I don't see why it has to be exposed by a different option.
My change here was just so it applied when necessary, because I was missing it when it was needed in some situations

Reply:
It may not need to be exposed but I don't think this option is by default visible in the cmake-gui and I want some option that does this visible.

Reply:
https://github.com/LLNL/GridDyn/pull/44/files#diff-af3b638bc2a3e6c650974192a53c7291R367 does that